### ISSUE TITLE
Deprecate warning

### ISSUE THREAD

**Issue by dusty-cjh** [torchbox/wagtail-markdown#122]
Hey master, maybe you can update your import path, I have got this notice for about a month:
```shell
RemovedInWagtail50Warning: Importing from wagtail.core.blocks is deprecated. Use wagtail.blocks instead. See https://docs.wagtail.org/en/stable/releases/3.0.html#changes-to-module-paths
  from wagtail.core.blocks import TextBlock
```

**Comment by zerolab** [comment]
Thank you for the prompt. It is on the list for this week (hopefully will get to it)
If you would like it faster, do feel free to submit a PR ;)

**Comment by zerolab** [comment]
fixed by #120
